webworker.js

```diff
@@ -40,7 +40,8 @@ async function startDatasette(settings) {
     toLoad.push(["content.db", "https://datasette.io/content.db"]);
   }
   self.pyodide = await loadPyodide({
-    indexURL: "https://cdn.jsdelivr.net/pyodide/v0.20.0/full/"
+    indexURL: "https://cdn.jsdelivr.net/pyodide/v0.22.1/full/",
+    fullStdLib: true
```

Comment:
I'd like to understand what difference `fullStdLib: true` makes.

Reply:
Starting with version [0.21.1](https://pyodide.org/en/stable/project/changelog.html#version-0-21-1), `fullStdLib` argument of `pyodide.loadPyodide` has changed to `false` by default. Previously automatically loaded packages are not anymore by default, such as `sqlite3`.

We can either enable `fullStdLib` like above or enable packages one by one using `pyodide.loadPackage` or `micropip.install`. The second option is more optimized but if users want to use certain Python packages from the standard library, they might not be loaded before hand and require a custom Datasette-Lite configuration.